test/testLuminous.js

```diff
@@ -139,43 +139,19 @@ describe("Configuration", () => {
 
   it("passes settings to Lightbox", () => {
     const anchor = document.querySelector(".test-anchor");
-    const settingsToMap = {
+    const settings = {
       namespace: "custom",
       sourceAttribute: "not-href",
       caption: "custom",
       includeImgixJSClass: true,
-      showCloseButton: {
-        value: false,
-        lightboxKey: "closeButtonEnabled"
-      }
+      closeButtonEnabled: false
     };
-    const isObject = v => typeof v === "object" && v != null;
-    const clientSettings = Object.keys(settingsToMap).reduce((p, key) => {
-      const valuePrimitiveOrObject = settingsToMap[key];
-      p[key] = isObject(valuePrimitiveOrObject)
-        ? valuePrimitiveOrObject.value
-        : valuePrimitiveOrObject;
-      return p;
-    }, {});
-
-    const lum = new Luminous(anchor, clientSettings);
-
-    Object.keys(settingsToMap).forEach(settingKey => {
-      const valuePrimitiveOrObject = settingsToMap[settingKey];
-      let expectedKey;
-      let expectedValue;
-      if (isObject(valuePrimitiveOrObject)) {
-        const valueConfig = valuePrimitiveOrObject;
-        expectedKey = valueConfig.lightboxKey || settingKey;
-        expectedValue =
-          "lightboxValue" in valueConfig
-            ? valueConfig.lightboxValue
-            : valueConfig.value;
-      } else {
-        expectedKey = settingKey;
-        expectedValue = valuePrimitiveOrObject;
-      }
-      expect(lum.lightbox.settings[expectedKey]).toBe(expectedValue);
+
+    const lum = new Luminous(anchor, settings);
+
+    Object.keys(settings).forEach(key => {
+      const value = settings[key];
+      expect(lum.lightbox.settings[key]).toBe(value);
```

Comment:
So, the previous setup allows the test case to test for configurations which have different external and internal names. Luminous accepts the option `showCloseButton: true | false`, but this is passed around internally as `closeButtonEnabled`.

---

As discussed in a previous comment, the changes in this PR break the public API for enabling/disabling the close button. This is why this change to the test case makes the test case pass - because the test case was meant to fail. 

There are two options here - we have to either find a way to keep the showCloseButton -> closeButtonEnabled mapping in the source code, or we rename all occurrences of closeButtonEnabled to showCloseButton. 

I'm happy to keep your changes to this test case if we do the second option above. Otherwise, we'll have to keep the old test code, as that will allow for the mapping of external prop -> internal variable.